apps/admin-x-framework/package.json

```diff
@@ -109,7 +109,8 @@
       "test:unit": {
         "dependsOn": [
           "test:unit",
-          "^build"
+          "^build",
+          "@tryghost/admin-x-design-system:test:unit"
```

Comment:
For some reason tests were flaking in CI without adding this — some kind of build/race condition I think